### ISSUE TITLE
Training: train adaboost

### ISSUE THREAD

**Issue by luca-m** [luca-m/emotime#6]
Review CvMLData, make it works or load directly feature matrix, transform it in a feature vector and manually prepare training set (and validation set)
